sway/tree/view.c

```diff
@@ -1244,13 +1244,9 @@ bool view_is_visible(struct sway_view *view) {
 			return false;
 		}
 	}
-	// Determine if view is nested inside a floating container which is sticky
-	struct sway_container *floater = view->container;
-	while (floater->parent) {
-		floater = floater->parent;
```

Comment:
This bit looks like an ad-hoc reimplementation of `container_is_floating_or_child`, so switching it to `container_is_sticky` seems valid.